client/src/app/(onboarding)/signup/page.tsx

```diff
@@ -1,3 +1,209 @@
+      formErrors.confirmPassword = "Passwords do not match";
+      isValid = false;
+    } else if (!confirmPassword) {
+      formErrors.confirmPassword = "Confirm password is required";
+      isValid = false;
+    }
+
+    setError(formErrors);
+    return isValid;
+  };
+
+  const handleSubmit = () => {
+    if (validateForm()) {
+      console.log("Email: ", email);
+      console.log("Password", password);
+    } else {
+      console.log("Validation failed");
+    }
+  };
+
+  return (
+    <div className="main-container">
+      {/* Let's get started section */}
+      <Typography variant="h4" sx={{ marginBottom: "20px" }}>
+        Let’s Get Started.
+      </Typography>
+
+      {/* Sign in with Google Button */}
+      <Button
+        className="borderline"
+        variant="outlined"
+        startIcon={
+          <img src="https://www.google.com/favicon.ico" alt="Google" />
+        }
+        sx={{
+          border: "4px solid #E0E4F2",
+          backgroundColor: "white",
+          width: "780px",
+          borderRadius: "8px",
+        }}
+      >
+        Sign up with Google
+      </Button>
+
+      {/* Divider with "Or" */}
+      <div className="divider">
+        <Divider
+          sx={{
+            width: "100px",
+            height: "1px",
+            backgroundColor: "#ccc",
+            marginRight: 2,
+          }}
+        />
+        <Typography
+          variant="h4"
+          sx={{
+            fontWeight: "bold",
+            fontSize: "1.25rem",
+            textAlign: "center",
+          }}
+        >
+          Or
+        </Typography>
+        <Divider
+          sx={{
+            width: "100px",
+            height: "1px",
+            backgroundColor: "#ccc",
+            marginLeft: 2,
+          }}
+        />
+      </div>
+
+      {/* Signup form */}
+      <div className="signup-box">
+        {/* Email field */}
+        <div className="text-field">
+          <TextField
+            fullWidth
+            label="Email"
+            placeholder="Enter your email"
+            required
+            error={!!error.email} // Check if there is an error
+            helperText={error.email} // Display error message
+            value={email}
+            onChange={(e) => setEmail(e.target.value)} // Update email state
+          />
+        </div>
+
+        {/* Password field */}
+        <div className="text-field">
+          <TextField
+            fullWidth
+            label="Password"
+            type="password" // Ensure this is password input
+            placeholder="Enter your password"
+            required
+            error={!!error.password} // Check if there is an error
+            helperText={error.password} // Display error message
+            value={password}
+            onChange={(e) => setPassword(e.target.value)} // Update password state
+          />
+        </div>
+
+        {/* Confirm Password field */}
+        <div className="text-field">
```

Comment:
why not just add the className to the TextField? I don't think the div is necessary. Same goes for the other textfields